src/storage/v2/vertex_accessor.hpp

```diff
@@ -35,6 +35,13 @@ class VertexAccessor final {
  private:
   friend class Storage;
 
+  int64_t HandleExpansionsWithoutEdgeTypes(edge_store &edges, query::HopsLimit *hops_limit,
```

Comment:
Let's rename `edges` to `result_edges` to be in sync with implementation and emphasize that this is returned value